src/docker-compose.yml

```diff
@@ -49,3 +49,18 @@ services:
     stdin_open: true
     tty: true
     command: ./run_node_watch.sh
+
+  pa11y:
+    build:
+      context: .
+      dockerfile: node.Dockerfile
+    cap_add:
+      - SYS_ADMIN
+    volumes:
+      - .:/app
+      - /app/node_modules
```

Comment:
This syntax (without the `:`) is unfamiliar to me. What does it do?

Reply:
It is an internal Docker volume that is intended to shadow the `node_modules` directory from the host so that it isn't mounted inside the container. I will add a better comment to this file.